libgamestream/http.c

```diff
@@ -67,6 +89,13 @@ int http_init(const char* keyDirectory) {
   curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, 0L);
   curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, _write_curl);
   curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1L);
+  curl_easy_setopt(curl, CURLOPT_SSL_CTX_FUNCTION, *sslctx_function);
```

Comment:
is this required? we already pass `CURLOPT_SSLCERT` and `CURLOPT_SSLKEY` (unlike the chrome port)